README.md

```diff
@@ -114,9 +114,10 @@ is provided by code that been generated from a model of the service.
 
 We need your help in making this SDK great. Please participate in the community and contribute to this effort by submitting issues, participating in discussion forums and submitting pull requests through the following channels.
 
-* [Contributions Guidelines](CONTRIBUTING.md)
+* [Contributions Guidelines](./Documents/CONTRIBUTING.md)
 * Articulate your feature request or upvote existing ones on our [Issues](https://github.com/aws/aws-iot-device-sdk-cpp-v2/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-request) page.
-* Submit [Issues](https://github.com/aws/aws-iot-device-sdk-cpp-v2/issues)
+* Create discussion questions [here](https://github.com/aws/aws-iot-device-sdk-cpp-v2/discussions)
+* Find a bug open an [issue](https://github.com/aws/aws-iot-device-sdk-cpp-v2/issues)
```

Comment:
Super minor: The wording on this seems a little strange to me, at least when I first read it. I would potentially reword it to something like to "[Open an issue] if you find a bug" or "Find a bug? Open an issue [here]".